content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -40,15 +40,29 @@ authenticate to Kiali.
 
 First, define the credentials you want to use as the Kiali username and passphrase:
 
+This will prompt you to provide Kiali Username.
+
 {{< text bash >}}
 $ KIALI_USERNAME=$(read -p 'Kiali Username: ' uval && echo -n $uval | base64)
+{{< /text >}}
+
+This will prompt you to provide Kiali Passphrase.
+
+{{< text bash >}}
 $ KIALI_PASSPHRASE=$(read -sp 'Kiali Passphrase: ' pval && echo -n $pval | base64)
 {{< /text >}}
 
 If you are using the Z Shell, `zsh`, use the following to define the credentials:
 
+This will prompt you to provide Kiali Username.
+
```

Comment:
Using _zsh_ on my Mac, the original single text works fine. I'm not sure if you found this failed using _zsh_ or if this is simply following what was done for _bash_. If you aren't seeing the original documented problem in _zsh_, I would revert back to a single copiable box.

Reply:
I didn't get chance to test it on Mac, but as you suggested I can revert it back for zsh to single copiable box if it's working fine.